docs/en/signal-hub-endpoint.rst

```diff
@@ -0,0 +1,169 @@
+
+In `Signal Hub pull`_ can be found non normative examples of a Signal Distribution requests and responses.
+
+If any error occurs during the request parsing, the response MUST adhere to the error format defined in `Signal Hub pull-yaml`_.
+
+.. note::
+  A complete OpenAPI Specification of the Signal Collection e-Service is available `Signal Hub pull-yaml`_.
+
+The Credential Issuer MUST implement the necessary logic to handle the Polling of the Signal Distribution e-Service endpoint, in doing this it has to consider the following aspects:
+
+  - Signals are queried and retrieved per PDND e-Service, meaning that the Credential Issuer MUST implement a poll cycle for each e-Service ID;
+  - the retention period of Signals in the Signal Hub is specified in `Signal Hub Guide`_. Signals older than the specified retention period are not available for retrieval;
+  - the Signal Hub does not keep trak of the last ``signalId`` notified to a specific Credential Issuer. Thus each Credential Issuer MUST keep track of the last ``signalId`` it has received for each PDND e-Service ID;
+  - the Signal Distribution e-Service endpoint returns Signals in batches. The maximal size the batch is specified in `Signal Hub Guide`_. The Signal Distribution e-Service will indicate if additional Signals are available for retrieval.
+
+Signals Processing
+^^^^^^^^^^^^^^^^^^^^
+After the Signals have been successfully recovered by the Credential Issuer, the latter MUST process them as follows:
+
+  - For each Signal, the Credential Issuer MUST check the ``SignalType`` value:
+    
+    - if the Signal ``SignalType`` is ``UPDATE``, it means that the status and/or value of the attribute associated with a Digest Credential need updates; instead,
+    - if the Signal ``SignalType`` is ``CREATE``, it means that the requested attributes of a specific Digital Credential are now available; 
+
+    If the ``objectId`` does not correspond to any valid identifier known to the Credential Issuer, the Signal MUST be ignored. If instead it corresponds to a known and valid identifier, the Credential Issuer MUST use the :ref:`authentic-source-endpoint:Get Attribute Claims` PDND endpoint of the Authentic Source to retrieve the updated information and in case apply the new status to the correspondent Credential.
+    
+    When the Signal has been processed, the Credential Issuer will either move to the next Signal and update its ``signalId`` counter; or, if there are no more Signals to process, it will resume the Pull cycle.
+
+.. warning::
+
+  Given Signal Hub's currently supported security patterns, if the Authentic Source requires the `AUDIT_REST_02` security pattern from the Credential Issuer, the latter MUST revoke the Digital Credential referenced in Signals with ``signalType`` ``UPDATE`` since it cannot contact the Authentic Source to retrieve the updated information without the User's consent.  
```

Comment:
```suggestion
  Given Signal Hub's currently supported security patterns, if the Authentic Source requires the `AUDIT_REST_02` security pattern from the Credential Issuer, the latter MUST revoke the Digital Credential referenced in Signals with ``signalType`` ``UPDATE`` since it cannot contact the Authentic Source to retrieve the updated information without having authenticated the User before.  
```